packages/frontend/src/redux/reducers/status/index.js

```diff
@@ -17,6 +17,19 @@ const initialState = {
 };
 
 const alertReducer = (state, { error, ready, payload, meta, type }) => {
+
+    // temporary solution to handle both `showAlert` and `showAlertToolkit`
```

Comment:
If this is temporary, what does the long-term solution look like?

Reply:
Added a more precise description to the file.

Generally `alertReducer` was designed to relay on `ready` param that is adding to action meta - this way we can recognize if the async action is finished. With redux toolkit and `createAsyncThunk` we have separate actions (`rejected` and `fulfilled`). So for now we need to use both `rejected` and `fulfilled` actions and `ready` param to recognize the action status, but finally, when we will be using redux-toolkit for all reducers, we will be able to rely on `rejected` and `fulfilled` types of actions only and we will refactor this whole reducer. There is also an option to rebuild the whole `globalAlert` related state, but it's on hold for now.